internal/services/mssql/mssql_database_vulnerability_assessment_rule_baseline_resource.go

```diff
@@ -153,7 +153,8 @@ func resourceMsSqlDatabaseVulnerabilityAssessmentRuleBaselineRead(d *pluginsdk.R
 	d.Set("rule_id", id.RuleName)
 	d.Set("baseline_name", id.BaselineName)
 
-	vulnerabilityAssessmentId := parse.NewServerVulnerabilityAssessmentID(id.SubscriptionId, id.ResourceGroup, id.ServerName, "default")
+	// TODO: replace "Default" with "default" after the issue(https://github.com/Azure/azure-rest-api-specs/issues/17192) is fixed.
+	vulnerabilityAssessmentId := parse.NewServerVulnerabilityAssessmentID(id.SubscriptionId, id.ResourceGroup, id.ServerName, "Default")
```

Comment:
since this is coming from the user input we can make this:

```suggestion
	vulnerabilityAssessmentId := parse.NewServerVulnerabilityAssessmentID(id.SubscriptionId, id.ResourceGroup, id.ServerName, id.DatabaseName, id.VulnerabilityAssessmentName)
```

which'll make it consistent with how this is handled above

Reply:
Thank you, Fixed.